### PR TITLE
Updated event cards to fetch event data from events website

### PR DESCRIPTION
**Pull request by ChristopherJMiller**

### CODE REVIEW

File: pages/index.jsx [hunk 1]
```diff
@@ -65,12 +64,24 @@ function Home() {
...
+  useEffect(() => {
+    if (loadingEvents) {
+      setLoadingEvents(false);
+      fetch('https://events.uf-ace.com/api/events/')
+        .then((response) => response.json())
+        .then(({ content }) => setEventData(content));
+    }
+  });
+
+  const eventCards = eventData.map((event) => (
```
Comment (ConnorJC3) [inline pages/index.jsx:79]: Login concerns? Or are we leaving this API open?

File: pages/index.jsx [hunk 2]
```diff
@@ -65,12 +64,24 @@ function Home() {
...
     <Event
       key={event.summary}
-      name={event.summary}
-      description={event.description}
-      datetime={event.datetime}
+      name={event.name}
+      description={event.description.replace(/<\/?\w+>/g, '').replace('&nbsp;', '').slice(0, 250)}
```
Comment (ConnorJC3) [inline pages/index.jsx]: No. Just no.

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]

**Comment by ConnorJC3** [comment]
Additional note: we need a time limiter on this, like don't show events from the past. As is this shows all events ever.

File: pages/index.jsx (hunk 1 above)
Comment (ChristopherJMiller) [reply pages/index.jsx:79]: This event data is public, that is by design

File: pages/index.jsx (hunk 2 above)
Comment (ChristopherJMiller) [reply pages/index.jsx]: Fair, will need to find a package to strip it more seemlessly

**Comment by ChristopherJMiller** [comment]
The events site is needing this feature as well. Most likely, the current request route in the future will only show upcoming events instead of all events, so this change will come to this site via the events site being updated.

File: pages/index.jsx (hunk 2 above)
Comment (ConnorJC3) [reply pages/index.jsx]: @ChristopherJMiller that's not my worry, my worry is why are we doing this in the first place?

Comment (ChristopherJMiller) [reply pages/index.jsx]: Resolved with most recent commit, changed to using short_description

**Review by ConnorJC3 — APPROVED** [review]
LGTM

**Pull request merged**